sections/oxygen_pre_deployment.md

```diff
@@ -39,173 +39,109 @@ When applying the tape, be sure that it is not in contact with any other metal p
-          
-           Set Passkey(1000)
-           
-           Do Calibrate
-The  Save command is automatically performed.
-
-9. Rinse well with distilled water and dry the sensor foil carefully and check how it works at air. 
-Save the data to check the sensor performance through time (a barometer is needed). 
-Value of oxygen saturation should be 100 % or higher.
-
-10. Put the sensor back into the 0 % water, the reading should drop to zero.
-
-11. Set back the Interval property to 1 second (or the desired sampling frequency).
-            
-            Set Passkey(1000)
-            
-            Set Interval(1)
-            
-            Save
-12. Type Get All command for saving the final sensor configuration for reference.
-
 ### In situ intercomparison in the tank during ballasting
-This is an extra in situ intercomparison to carry out if  access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before) .
+This is an extra in situ intercomparison to carry out if access to the tank while ballasting the glider is possible and the ballasting is close in time to the deployment (no more than 1-2 days before) (reference PLOCAN).
 
 Materials: Silicon tube for sampling, multiparameter sonde, BOD bottles, Winkler reagents, pipettes and tips (or a bottle-top dispenser for reagent bottles), titration material (buretes or titrator).
 
-1. The sensor should stay for at least 24 h submerged  in water to make the membrane wet. If the sensor is already mounted in the glider, use a wet sponge. 
-*NOTE: Keep the sensor in the dark all time.*
+1. The sensor should be submerged overnight in water to ensure the foil is wet. If the sensor is already mounted in the glider, use a wet sponge or rubber lens cap. Keep the sensor in the dark.
 4. Once the glider is in the ballasting tanks, place the silicon tube for sampling near the sensor.
-5. Once the sensor measurements are stable (variations in the measurements are not higher than the precision/resolution of the sensor), start collecting Winkler samples.  Take samples every 5-10 minutes, in total 4-6 samples will be required. *NOTE: Record the time we collect a sample for Winkler titration.*
-6. A multiparameter sonde with a DO sensor whose precision is less than 0.1 % can be measuring in the tank near the glider’s sensors (record at least values for Temperature, Conductivity and DO). 
+5. Once the sensor measurements are stable (variations in the measurements are not higher than the precision/resolution of the sensor), start sampling water for Winkler analysis. Take paired samples every 5-10 minutes and record the time taken. 4 to 6 samples should be taken in total.
+6. A multiparameter sonde with a oxygen sensor whose precision is less than 0.1 % can be measured in the tank near the glider’s sensors (record at least values for Temperature, Conductivity and DO). 
```

Comment:
should we take this out? I think to introduce another instrument which has to be calibrated too to be helpful just creates more work or?